receipe/emailverification/emailverificationfunctions.go

```diff
@@ -22,16 +23,17 @@ func createAndSendCustomEmail(appInfo supertokens.NormalisedAppinfo) func(user U
 		}
 		jsonData, err := json.Marshal(data)
 		if err != nil {
-			return err
+			fmt.Println(err) // todo: handle error
```

Comment:
You can remove this "todo: handle error" since simply doing return is exactly what's needed.